scio-core/src/main/java/com/spotify/scio/options/ScioOptions.java

```diff
@@ -39,6 +39,10 @@
   boolean isBlocking();
   void setBlocking(boolean value);
 
+  @Description("Time period in scala.concurrent.duration.Duration style to wait for job completion")
+  String getJobTimeout();
+  void setJobTimeout(String value);
```

Comment:
🎨 I think this name makes the arg meaning vague. Maybe `blockFor` or even just `await`?

Reply:
I kind of like `blockFor` since I would expect `await` to be a boolean type